src/SdkCommon/ClientRuntime/ClientRuntime/TokenCredentials.cs

```diff
@@ -112,8 +112,8 @@ public TokenCredentials(ITokenProvider tokenProvider, string tenantId, string ca
                 throw new InvalidOperationException(Resources.TokenProviderCannotBeNull);
             }
 
-            request.Headers.Authorization = await TokenProvider.GetAuthenticationHeaderAsync(cancellationToken);
-            await base.ProcessHttpRequestAsync(request, cancellationToken);
+            request.Headers.Authorization = await TokenProvider.GetAuthenticationHeaderAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
This is not backward compatible, and has the potential to cause issues with consumers using this library within an ASP.NET project.  The context should be retained.  Please see this answer: https://stackoverflow.com/questions/40198816/is-there-any-danger-in-using-configureawaitfalse-in-webapi-or-mvc-controllers/40199432